### PR TITLE
Easy Pick: updated README.md with installation instructions; added .gitignore file

### PR DESCRIPTION
**Pull request by tintinthong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,34 @@ A library for probabilistic programming in Haskell using probability monads. The
 
 For the code associated with Haskell2015 paper "Practical Probabilistic Programming with Monads" see the `haskell2015` branch.
 
+## Installation (using Stack)
+
+Ensure stack is already installed by following these [instructions](https://docs.hashttps://docs.haskellstack.org/en/stable/install_and_upgrade/)
+
+To fork repo:
```
Comment (dschrempf) [inline README.md]: To _clone the_ repo:

File: README.md [hunk 2]
```diff
@@ -6,6 +6,34 @@ A library for probabilistic programming in Haskell using probability monads. The
...
+stack build
+```
+
+To test the code:
+
+```
+stack test
+```
+
+To open interactive session:
```
Comment (dschrempf) [inline README.md:31]: To open _an_ interactive session:

**Review by dschrempf — COMMENTED** [review]
Is there a reason why this pull request is not merged? I think it is important to explain how this library is used. Also, is there a reason why it is not on `hackage`? Thanks!

**Comment by adscib** [comment]
Sorry, forgot to merge this. Thanks for the contribution!

Official release coming soon.

**Pull request merged**